agent/util-scripts/pbench-postprocess-tools

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; tab-width: 8 -*-
```

Comment:
indent-tabs-mode nil? shouldn't offsets and indents be 4?

Reply:
I am not sure why we want the `indent-tabs-mode` to be `nil` for this file, when all other files are pretty much using `t`.  Also, we are just changing the tab-width to be 8, but the original indentation before this change was already 8, so why do we want to change it to 4 in this commit?